.github/workflows/rust.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           toolchain: ${{ env.NIGHTLY_RUST_VERSION }}
           components: rustfmt, clippy
           target: wasm32-unknown-unknown
+          rustflags: ${{ contains(matrix.command, 'wasm32-unknown-unknown') && '--cfg getrandom_backend="wasm_js" -D warnings' || '-D warnings' }}
```

Comment:
Shouldn't this be handled by cargo config? We have f9307f0755ae3d415c48cc044cbbd326a01255a4.